blog/tests/test_views.py

```diff
@@ -1,9 +1,32 @@
 from django.test import TestCase
 from rest_framework.test import APITestCase
+from blog.models import Stori, Category
+import json
 
 # Create your tests here.
 
 # class TestStoriList(APITestCase):
     
 #     def test_storilist(self):
-#         self.client
\ No newline at end of file
+#         self.client
+
+class TestStoriList(TestCase):
+    def setUp(self):
+        self.test_category = Category.objects.create(name="Test Category")
+
+        Stori.objects.create(title="Test 1", slug="test-1", description="Test 1", content="Story 1", category=self.test_category)
+        Stori.objects.create(title="Test 2", slug="test-2", description="Test 2", content="Story 2", status="Published", category=self.test_category)
+
+    """
+    Test the /mastori/ endpoint to ascertain whether
+    only published blogs are returned
+    """
+    def test_only_published_stori_returned(self):
+        response = self.client.get("/mastori/")
```

Comment:
I would advise using the reverse function which takes the end-point name instead of hardcoding the endpoint to avoid test breaking in future incase the endpoint is changed. example 
`response = self.client.get(reverse("name))`